.circleci/config.yml

```diff
@@ -115,69 +120,59 @@ only_release: &only_release
 workflows:
   default:
     jobs:
-      - horizon/block:
-          <<: *only_release
-          context: horizon
-          project_id: 11
+      # Main build
+      - artsy-remote-docker/build:
+          <<: *not_staging_or_release
+          context: hokusai
+          name: build
+          pre-steps:
+            - run:
+                command: echo 'export BUILD_TARGET="builder"; export DOCKER_BUILDKIT=1; export BUILDKIT_PROGRESS=plain; export COMPOSE_DOCKER_CLI_BUILD=1;' >> $BASH_ENV
```

Comment:
This isn't too bad.

Reply:
We don't "need" to do this. It enables `buildkit`, makes the output greppable, and tells `docker compose` to use the docker cli. If this works well, it would make sense to add this to the orb.

Reply:
Is there a minimum version of Docker that we need to use for buildkit support?

Reply:
The version that we are on is the minimum supported version.